components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -139,11 +140,18 @@ export default function (props: { project?: Project; isAdminDashboard?: boolean
         return -1;
     };
 
-    const triggerPrebuild = (branchName: string | null) => {
+    const runPrebuild = async (branchName: string | null) => {
         if (!project) {
             return;
         }
-        getGitpodService().server.triggerPrebuild(project.id, branchName);
+        setIsRunningPrebuild(true);
+        try {
+            await getGitpodService().server.triggerPrebuild(project.id, branchName);
+        } catch (error) {
+            console.error("Could not run prebuild", error);
```

Comment:
It would be nice to add a plain `alert` here at least.

Reply:
True, I agree with the sentiment 👍 (but maybe more an `<Alert>` component overalls, to properly show an error message in the UI)